collectors/prometheus.go

```diff
@@ -94,11 +96,21 @@ func NewPrometheusExporter(cfg *PrometheusConfig, lnd *lndclient.LndServices,
 
 	htlcMonitor := newHtlcMonitor(lnd.Router, errChan)
 
+	// Setup signalling so SIGTERM || SIGINT will stop the cache goroutine
+	// from ChannelsCollector
+	sigs := make(chan os.Signal, 1)
```

Comment:
This can now all be removed since we already pass in the `quitChan` (which will be closed on these signals with `>-interceptor.ShutdownChannel()`).